engine/src/flutter/shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -781,6 +781,18 @@ - (void)accessibilityElementDidLoseFocus {
   }
 }
 
+- (BOOL)accessibilityRespondsToUserInteraction {
+  if ((self.node.actions & flutter::~kSystemActions) != 0) {
```

Comment:
I am surprise this works, shouldn't this be `~flutter::kSystemActions`

Reply:
it doesnt work cuz the tests are failing 😅. My bad I was copy and pasting my changes cuz I had to force push to an earlier commit